src/backend/utils.ts

```diff
@@ -1348,10 +1349,57 @@ interface ExtractOptions {
 }
 
 async function extractFiles({ path, destination, strip = 0 }: ExtractOptions) {
-  return decompress(path, destination, {
-    plugins: [decompressTargz(), decompressTarxz()],
-    strip
-  })
+  if (!isSnap && (path.endsWith('.tar.xz') || path.endsWith('.tar.gz'))) {
+    try {
+      await extractNative(path, destination, strip)
+    } catch (error) {
+      logError(`Error: ${error}`, LogPrefix.Backend)
```

Comment:
```suggestion
      logError(['Error:', error], LogPrefix.Backend)
```
Passing the error directly to the logging function makes it print more information if available